.circleci/config.yml

```
@@ -227,6 +227,50 @@ jobs:
           name: generate doxygen documentation
           command: inv -e rtloader.generate-doc
 
+  macos_tests:
+    macos:
+      xcode: 10.1.0
+    environment:
+      BUILDIMAGES_VERSION: "a8226a8e78774afad285e7372fe8c6c179340dd9" # datadog-agent-buildimages commit to fetch to get the MacOS scripts
+    working_directory: ~/go/src/github.com/DataDog/datadog-agent
+    steps:
+      - checkout
+      - run:
+          name: Setup runner
+          command: |
+            bash -c "$(curl -L https://raw.githubusercontent.com/DataDog/datadog-agent-buildimages/$BUILDIMAGES_VERSION/macos/builder_setup.sh)"
+      - run:
+          name: Install python deps & clang-format
+          command: |
+            brew install clang-format
+            pip install -r requirements.txt
```

```suggestion
            python3 -m pip install -r requirements.txt
```
(or `python2` if it's Python 2)